From f86deeea77a428f425df9a5f92134330e8efa53a Mon Sep 17 00:00:00 2001
From: "Diwakar, Praveen" <praveen.diwakar@intel.com>
Date: Mon, 5 Dec 2016 16:31:05 +0530
Subject: [PATCH 4046/4301] ASoC: Intel: board: Add Glenview machine id

commit 8bb4ddd853093ea25d3d20e2ba4e4d585af2891f from
git://git@github.com:01org/linux-intel-4.9.git

Glenview also features wm8281 codec, so use the same machine driver
for glenview as well.

Change-Id: I09084f8e99ee6d666791aa5e80a2a8321ec25231
Signed-off-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/12272
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/boards/cnl_wm8281.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/soc/intel/boards/cnl_wm8281.c b/sound/soc/intel/boards/cnl_wm8281.c
index c85b6e0..3219724 100644
--- a/sound/soc/intel/boards/cnl_wm8281.c
+++ b/sound/soc/intel/boards/cnl_wm8281.c
@@ -665,6 +665,7 @@ const struct dev_pm_ops snd_cnl_florida_mc_pm_ops = {
 
 static const struct platform_device_id cnl_board_ids[] = {
 	{ .name = "cnl_florida" },
+	{ .name = "glv_wm8281" },
 	{ .name = "icl_wm8281" },
 	{ }
 };
-- 
1.7.5.4

